main.py

```diff
@@ -3,3 +3,11 @@
 
 def detectar_nulos(df):
     return df.isnull().sum()
+
+def calcular_promedios(df):
+    return df.mean()
+
+def calcular_tipos(df):
+    for col in df.iteritems():
+        print(col.dtype)
```

Comment:
Considero que esa implementación no es eficiente, sugiero cambiarla:

```suggestion
def calcular_tipos(df):
    print(df.dtypes)
```